app/build.gradle

```diff
@@ -150,10 +157,13 @@ dependencies {
     testRuntimeOnly "org.junit.jupiter:junit-jupiter-engine:$junit5_version"
     testRuntimeOnly "org.junit.vintage:junit-vintage-engine:$junit5_version"
 
+    androidTestImplementation rootProject.ext.aosp_libs
     androidTestImplementation 'androidx.test:runner:1.4.0'
     androidTestImplementation 'androidx.test:rules:1.4.0'
     androidTestImplementation 'androidx.test.ext:junit:1.1.3'
     androidTestImplementation "io.mockk:mockk-android:$mockk_version"
+
+    androidTestImplementation 'com.kaspersky.android-components:kaspresso:1.5.3'
```

Comment:
What is this for? Does this get used anywhere?

Reply:
That's [this library](https://github.com/KasperskyLab/Kaspresso) which we use as a [base test class](https://github.com/seedvault-app/seedvault/blob/259ec52054cba4cd5bdbd35832717266829426e6/app/src/androidTest/java/com/stevesoltys/seedvault/e2e/LargeTestBase.kt#L17) and for [some screens](https://github.com/seedvault-app/seedvault/blob/259ec52054cba4cd5bdbd35832717266829426e6/app/src/androidTest/java/com/stevesoltys/seedvault/e2e/screen/impl/RecoveryCodeScreen.kt).

Reply:
Removed this library since we can get by without it and it wasn't providing much value.